valkyrie.drush.inc

```diff
@@ -121,7 +121,11 @@ function valkyrie_drush_command() {
     'aliases' => array('vup'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUSH, // No bootstrap at all.
   );
-
+  $items['valkyrie-upgrade'] = array(
+    'description' => 'Upgrade global Valkyrie extension and all of the submodules.',
+    'aliases' => array('vug'),
```

Comment:
The default `drush valkyrie-self-upgrade` behaviour **should** simply be to call `drush dl`. Presumably, most people will install from the release (maybe via a one-liner, see #76), so upgrading to the current release is the most [reasonable expectation](https://en.wikipedia.org/wiki/Principle_of_least_astonishment). And yes, it is somewhat redundant. However, it would give us an opportunity to check on Vagrant and Virtualbox versions, for example, if we up those in a future release. I'm not suggesting that we do that now, but it's something to consider.

It *also* allows us to switch to git, which is the point of #22. This'll allow users to easily test feature branches, bug fixes, or just live on the bleeding edge. However, once we're using the git version, we should be able to switch back to releases, if more stability is desired. We can check for the presence of `.git/` to determine whether the default should be `drush dl valkyrie`, or `git pull && git submodule update --recursive`.

Eventually, it might be nice to provide an `--update-projects` option, that would look at the machine output of `vagrant global-status` and update all the valkyrie-based projects that it finds automatically.